src/client/Client.cc

```diff
@@ -6561,15 +6562,15 @@ void Client::tick()
 {
   ldout(cct, 20) << "tick" << dendl;
 
-  utime_t now = ceph_clock_now();
+  ceph::coarse_mono_time now = ceph::coarse_mono_clock::now();
 
   /*
    * If the mount() is not finished
    */
   if (is_mounting() && !mds_requests.empty()) {
     MetaRequest *req = mds_requests.begin()->second;
 
-    if (req->op_stamp + cct->_conf->client_mount_timeout < now) {
+    if (req->coarse_stamp + (cct->_conf.get_val<std::chrono::seconds>("client_mount_timeout")) < now) {
```

Comment:
you should use the `mount_timeout` directly, no need to get the val from `_conf` every time.

Reply:
Done